index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>CSD120 Malcolm - Larmond</title>
 </head>
-<body>Hello,world!
-    
+<body>
+    Hello,world!
```

Comment:
Typically, if the opening and closing tags are on separate lines then the content is also on its own line.